templates/project/.github/workflows/qa.yaml.twig

```diff
@@ -70,7 +70,6 @@ jobs:
             - name: Psalm
               run: vendor/bin/psalm --no-progress --show-info=false --stats --output-format=github --threads=$(nproc) --shepherd --php-version={{ branch.targetPhpVersion.toString }}
 {% if project.rector %}
-
```

Comment:
We add a line between every step. Might be better to keep this line and add one before
```
psalm:
```

WDYT ?

Reply:
Yes we can do that instead.

Reply:
Lets only enable rector for block bundle for now. Later we will fix this empty line issue